### ISSUE TITLE
Issue installing python environment

### ISSUE THREAD

**Issue by ecm200** [matterport/Mask_RCNN#2041]
Hi.

I have been trying to get Mask RCNN package installed into a python environment and I have been really struggling to get the package versions to work.

Does anyone have a package versions list that definitely works with repository which I can replicate?
What version of CUDA does this environment use?

I have managed to get installs of tensorflow-gpu v1.13.1 and v1.14 and v1.15 installed with the other dependencies. I have found:

v1.13.1 will only run on the cpu.

> v1.14 errors at the model fit execution complaining regarding .metrics_tensor doesn't exist.

**Comment by suchiz** [comment]
Working fine on:
Windows 10
python 3.6.10
Tensorflow-gpu 1.13.1
Keras 2.2.5
Cuda 10.0
Cudnn 7.6.5.32

**Comment by Akshay-Ijantkar** [comment]
@suchiz THANKS  ALOT buddy! You saved me
